src/Transformations.jl

```diff
@@ -210,6 +210,14 @@ function normalize!(data, normalization::ZScore)
     return nothing
 end
 
+function inverse_normalize!(data, normalization::ZScore)
```

Comment:
What happens when `inverse_normalize!` is called with another normalization?

Reply:
Perhaps change this to `denormalize!` and add a docstring

Reply:
Sure